package.json

```diff
@@ -18,9 +18,9 @@
     "test:ci": "npm run lint && npm run unit -- --cov --coverage-report=lcovonly && npm run test:typescript",
     "test:report": "npm run lint && npm run unit:report && npm run test:typescript",
     "test:typescript": "tsd",
-    "unit": "tap -J test/*.test.js test/*/*.test.js",
+    "unit": "tap --before=test/before-tests.js -J test/*.test.js test/*/*.test.js",
     "unit:junit": "tap-mocha-reporter xunit < out.tap > test/junit-testresults.xml",
-    "unit:report": "tap -J test/*.test.js test/*/*.test.js --cov --coverage-report=html --coverage-report=cobertura | tee out.tap"
+    "unit:report": "tap --before=test/before-tests.js -J test/*.test.js test/*/*.test.js --cov --coverage-report=html --coverage-report=cobertura | tee out.tap"
```

Comment:
Can we move the --before setting inside the `.taprc`?

Reply:
I would rather not see `--before` used at all. It will make usage of `tap` as the test runner a requirement, i.e. we would not be able to `node test/something.test.js`.

Reply:
We could move this in `pretest` script and a `test()` case on the `https.test.js` that check if the cert files are missing and show a warning "did you run `npm test` to generate the local https certificates?"

Reply:
Why can't they be generated at the start of the test suite before the tests actually run?